gcp/volume.go

```diff
@@ -141,8 +142,48 @@ type mountPoints struct {
 }
 
 func (c *Client) getVolumeByID(volume volumeRequest) (volumeResult, error) {
+	var baseURL string
+	var originalID string = ""
+
+	// terraform import will specify volumeID.
+	// Issue is, that volumeID is unqiue per region, but might exist in different regions in same project.
+	// For that case, we will support a special ID format for terraform import ADDR ID.
+	// ID = <volumeID>:<region>
```

Comment:
It's a nitpick
Would   <region>:<volumeID>   be more natural?

I don't really have a preference.

Reply:
Sorry, I don't understand. Do you mean replace "will" by "would"? Please clarify.

Reply:
It's a nitpick
Would   `<region>:<volumeID>`   be more natural?

I don't really have a preference.

Reply:
Ahh, I understand. You are right, it would be more natural. But I still like to stick with the current way, because it better reflects the workflow. As a suer I would start out by specifying the volumeID only. This will work in nearly all cases. Only for the unlikely case that we really have a volumeID collision, one would add the region. A user would do that by simply get the existing command from command history. The cursor would be already at the end of the line, so adding the region at the end is more convenient.

Reply:
good for me.  Thanks!